lib/suma/admin_api/search.rb

```diff
@@ -8,6 +8,14 @@ class Suma::AdminAPI::Search < Suma::AdminAPI::V1
   include Suma::AdminAPI::Entities
 
   resource :search do
+    helpers do
+      def ds_search_or_order_by_name(ds, params)
```

Comment:
This is one of those things where the code is the same out of coincidence- if you rename something to not search on `name` this method would have to be rewritten. Instead let's refactor this to not be brittle/coincidentally DRY.